app.js

```diff
@@ -138,26 +138,36 @@ router.get('/account/:name', async ctx => {
 
 /**
  * Create a new account. Generate a throw away account id (UUID).
```

Comment:
Let's better do API docs in a separate doc using https://apiblueprint.org
I think ultimately it's far more clear if you can see possible example responses, etc

Reply:
Agreed. It will be easier to do more robust docs from these breadcrumbs though, so let's keep them for now.